hack/lib/serverless.bash

```diff
@@ -212,7 +212,7 @@ function deploy_knativeeventing_cr {
 
   # Install Knative Eventing
   cat <<EOF > "$eventing_cr"
-apiVersion: operator.knative.dev/v1alpha1
+apiVersion: operator.knative.dev/v1beta1
```

Comment:
From a test perspective, I'd stick with v1alpha1 for at least one release

Reply:
Yes makes sense, I am bit eager to test the new ones for now.